crates/walrus-core/src/by_axis.rs

```diff
@@ -183,7 +183,7 @@ impl From<Axis> for ByAxis<(), ()> {
 /// and `Secondary` variants.
 #[macro_export]
 macro_rules! flat_map {
-    ($by_axis:expr, $($fn_once:tt)*) => {
+    ($by_axis:expr_2021, $($fn_once:tt)*) => {
```

Comment:
Suggestion: Keep the `expr` everywhere unless we really need the old matching behavior. See also the [migration recommendations](https://doc.rust-lang.org/edition-guide/rust-2024/macro-fragment-specifiers.html#migration).

Reply:
Done.